api/config.py

```diff
@@ -0,0 +1,6 @@
+import tomli
```

Comment:
I would be against using this way of configuration, .env files are accepted standard way of configuring the applications in almost any language + you need a way to configure the app from environment variables as described here: https://12factor.net/config . 

If we need to structure the environment variables, you can still do it with .env files, just put comments above each section (# AZURE etc.)

Reply:
alright I changed it